sdk/servicebus/Azure.Messaging.ServiceBus/tests/Processor/ProcessorLiveTests.cs

```diff
@@ -236,7 +239,7 @@ async Task ProcessMessage(ProcessMessageEventArgs args)
                 }
                 await Task.WhenAll(completionSources.Select(source => source.Task));
                 Assert.IsTrue(processor.IsProcessing);
-                await processor.StopProcessingAsync();
+                await processor.CloseAsync();
```

Comment:
This seems odd;  don't we normally expect the previous pattern in customer code?    Just calling stop and then close/dispose when the application is exiting?

Reply:
Yeah, I may have went overboard with the replacements here. I updated a few tests where we were checking that all of the messages were gone from the queue because otherwise in the case of sessions, the session would remain locked when just calling StopProcessingAsync.